src/lib/LoginFile.js

```diff
@@ -115,11 +115,11 @@ class LoginFile {
         /* eslint-disable no-multi-spaces */
         const $canvas = QrEncoder.render({
             text: encodedSecret,
-            radius: 0.7,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
+            radius: 0.5,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
             ecLevel: 'M',   // the error-correction level to M. Thus we reduce the radius from .8 to .7
             fill: 'white',  // to reduce scanning issues.
             background: 'transparent',
-            size: Math.min(240, (window.innerWidth - 64)),
+            size: LoginFile.QR_SIZE,
         });
         /* eslint-enable no-multi-spaces */
         if (!$canvas) throw new Error('Cannot draw QR code');
```

Comment:
Why does that canvas get appended to a div?

Reply:
It doesn't, apparently, but I did not change that in this PR :stuck_out_tongue: 
Removed it now ;)